src/Answer.tsx

```diff
@@ -0,0 +1,110 @@
+/* eslint @typescript-eslint/no-explicit-any: "off" */
+/* eslint @typescript-eslint/no-unsafe-assignment: "off" */
+import * as React from "react";
+import prettyTitle from "lodash.startcase";
+import styled from "@emotion/styled";
+
+type AnswerProps = React.PropsWithChildren<{
+  title: string;
+  details: any;
+}>;
+
+export default function Answer(
+  props: AnswerProps
+): React.ReactElement<AnswerProps, any> {
+  const Item = styled.div`
+    padding: 1em 0;
+    border-bottom: 1px solid #00000022;
+    page-break-inside: avoid;
+  `;
+
+  const Title = styled.p`
+    margin: 0 0 1em 0;
+    font-weight: bold;
+  `;
+
+  const checkAnswerProps = (props: object) => {
+    return Object.hasOwn(props, "title") && Object.hasOwn(props, "details");
+  };
+
+  if (checkAnswerProps(props)) {
+    return (
+      <Item>
+        <Title>
+          {
+            // eslint-disable-next-line @typescript-eslint/no-unsafe-call
+            prettyTitle(props.title)
+          }
+        </Title>
+        <Details data={props.details} />
+      </Item>
+    );
+  }
+  return <p>[error]</p>;
+}
+
+function Details(props: { data: any }): React.ReactElement<any, any> {
+  const Empty = styled.span`
+    color: #00000033;
+  `;
+  const { data } = props;
+  if (!data) {
+    return <Empty>[empty]</Empty>;
+  }
+  if (typeof data === "string" || typeof data === "number") {
+    return <span>{data}</span>;
+  }
+  if (Array.isArray(data)) {
+    return List(data);
+  }
+  if (typeof data === "object") {
+    // eslint-disable-next-line @typescript-eslint/no-unsafe-argument
+    return Tree(data);
+  }
```

Comment:
I recognise this logic...! 😉 👍 https://github.com/theopensystemslab/planx-new/pull/1032/files#diff-babdae6181c0ecf3140ce91277e3b570ab98cc7bfe8aee325cc9478c808c1c98R206-R213